src/recipe.cpp

```diff
@@ -314,6 +314,21 @@ void recipe::load( const JsonObject &jo, const std::string &src )
         assign( jo, "description", description, strict );
         assign( jo, "reversible", reversible, strict );
 
+        if( jo.has_member( "uncraft_time" ) ) {
+            if( ! reversible ) {
+                jo.throw_error( "Recipe cannot have uncraft_time yet not be reversible" );
+            }
+
+            if( jo.has_int( "uncraft_time" ) ) {
+                // Time in moves
+                assign( jo, "uncraft_time", uncraft_time, strict );
```

Comment:
I think we were moving away from ambiguous integers as time specifiers (are they moves? seconds? or maybe hours?). 
I'd suggest we not introduce new sources of it.

Being able to specify the moves is only *maybe* useful for debugging C++ issues, but at that point you can just override the moves required in C++ proper.

Reply:
Yea as long as we don't have sub-second uncrafts I'd prefer using time units.

Reply:
Excellent! I am delighted for this to be time-unit only!